src/gui/guiFormSpecMenu.cpp

```diff
@@ -2794,6 +2795,11 @@ void GUIFormSpecMenu::parseModel(parserData *data, const std::string &element)
 	e->enableContinuousRotation(inf_rotation);
 	e->enableMouseControl(mousectrl);
 
+	const s32 frame_loop_begin = stoi(frame_loop[0]);
+	const s32 frame_loop_end = frame_loop.size() == 1 ? 0x7FFFFFFF : stoi(frame_loop[1]);
```

Comment:
It really isn't necessary for these to be const.  Const is useful for e.g. const references as a function parameter, but don't go overboard.